Author: Fathi Boudra<fabo@debian.org>
Description: Append "-kde" to svn-clean like KDE3.
 svn-clean script is provided by subversion-tools.
Bug-Debian: http://bugs.debian.org/461095

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -76,7 +76,6 @@
   cvs-clean
   cvsblame
   colorsvn
-  svn-clean
   svnlastchange
   svnversions
   extractrc
@@ -99,3 +98,5 @@
   DESTINATION ${BIN_INSTALL_DIR})
 
 feature_summary(WHAT ALL FATAL_ON_MISSING_REQUIRED_PACKAGES)
+
+install(PROGRAMS svn-clean DESTINATION ${BIN_INSTALL_DIR} RENAME svn-clean-kde)
